### ISSUE TITLE
Firefox still requires prefixes for `fit-content` as a width or height

### ISSUE THREAD

**Issue by dak** [postcss/autoprefixer#1207]
`max-content` and `min-content` support was only added in Firefox 66, and `fit-content` is still behind the `-moz-` prefix.

**Comment by ai** [comment]
Do we have any Can I Use note about it to use this data in Autoprefixer?

**Comment by ai** [comment]
Firefox has no special note on Can I Use https://caniuse.com/#search=fit-content

Can you give me some docs about `fit-content`?

It will be even better if you will open PR at [caniuse](https://github.com/Fyrd/caniuse/). Just mention me in PR, I will help.

**Comment by Dan503** [comment]
You can find full browser compatibility notes on MDN https://developer.mozilla.org/en-US/docs/Web/CSS/width#Browser_compatibility

**Comment by ai** [comment]
Perfect. Can somebody help me with Can I Use and I will do Autoprefixer’s part of the feature?

**Comment by Dan503** [comment]
You will notice that Opera needs a `-webkit-` vendor prefix as well by the looks of it.

**Comment by nylen** [comment]
I was having the same issue, but changing from "last 2 firefox versions" to "Firefox >= 60" (the current ESR version) fixed it.

CanIUse site is updated appropriately:

![2019-07-13T23 51 38-05 00](https://user-images.githubusercontent.com/227022/61179500-22250c00-a5f3-11e9-86d7-eb97e25234bc.png)

Can this be closed?

**Comment by ai** [comment]
I need some help here. What is input CSS, current output and expected output?

(Sorry, too many open sources to look deeper)

**Comment by kelly-tock** [comment]
the setting above worked for me as well. strange as it seems like the last 2 versions should be fine as well.

**Comment by butsalt** [comment]
@ai Accroding to [caniuse](https://caniuse.com/#search=fit-content), `fit-content` should always be prefixed with `-moz-` for `Firefox` until now. But in the `autoprefixer`, `fit-content` has been unprefixed since `Firefix >= 66`.


autoprefixer version: `9.7.0`


browsers: `Firefox >=66`


input:
```css
.main {
  width: fit-content;
}
```


current output:
```css
.main {
  width: fit-content;
}
```


expected output:
```css
.main {
  width: -moz-fit-content;
  width: fit-content;
}
```

**Comment by ai** [comment]
@butsalt thanks. I will try to fix it tomorrow.

**Comment by ai** [comment]
Fixed 7869a83

**Comment by ai** [comment]
Nope, we can’t use [fit-content](https://caniuse.com/#feat=mdn-css_properties_width_fit-content) Can I use data, since it is MDN data and can’t be found in `caiuse-lite` npm package

**Comment by ai** [comment]
Fixed 89d723c

**Comment by ai** [comment]
I will release this fix when https://github.com/postcss/autoprefixer/pull/1260 will be merged

**Comment by ai** [comment]
Released in 9.7.1
